Makefile

```diff
@@ -48,3 +48,8 @@ cross: $(GOX)
 # Docker container
 container: static
 	docker build -t grafana/grizzly .
+
+docs:
```

Comment:
`make docs` sounds like it generates docs (as make's default function is to make things). I'd call it `serve-docs`